Add standard CloudWatch instance alarms for the OpenVPN instance

jsf9k opened the pull request:
## 🗣 Description ##

This pull request adds a set of standard CloudWatch instance alarms for the OpenVPN instance.

## 💭 Motivation and context ##

These alarms will notify us via email if any of the following should occur:
- Either of an instance's status checks (instance or system) fail
- Any IMDSv1 requests succeed
- CPU, memory, or disk utilization is high
- Any packets are queued and/or dropped because the a networking limitation was exceeded for the instance

## 🧪 Testing ##

All automated testing passes.  This code is also already applied to the Shared Services account in our COOL staging environment.

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All relevant repo and/or project documentation has been updated to reflect the changes in this PR.
- [x] All new and existing tests pass.

## ✅ Pre-merge checklist ##

- [x] Revert dependencies to default branches.

jsf9k committed:
Add standard CW alarms for the OpenVPN instance

jsf9k committed:
Add @mcdonnnj as a codeowner

dav3r reviewed (approved):
👍  Don't forget to also update the `terraform-docs` stuff in the README after you switch back to the default branch.

jsf9k commented:
> +1 Don't forget to also update the `terraform-docs` stuff in the README after you switch back to the default branch.

Thanks, that is a good reminder!

jsf9k committed:
Use the default branch of cisagov/instance-cw-alarms-tf-module

The branch we were using was merged into the default branch via
cisagov/instance-cw-alarms-tf-module#5.
